docs/contribute/index.md

```diff
@@ -10,20 +10,114 @@ Whether you're a technical writer or you've only edited Elastic docs once or twi
 
 * For **simple bugfixes and enhancements** --> [contribute on the web](on-the-web.md)
 * For **complex or multi-page updates** --> [Contribute locally](locally.md)
 
+## Where to find docs source files
+
+Understand where your content of interest now sits in the new docs information architecture and the repos where source pages live.
+
+### Kibana
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **explore-analyze**: Discover, Dashboards, Visualizations, Reporting, Alerting, dev tools...  
+* **deploy-manage**: Stack management (Spaces, user management, remote clusters...)  
+* **troubleshooting**: troubleshooting pages
+
+#### What is staying in the Kibana repo?
+
+* Reference content, anything that is or could be auto-generated: Settings, syntax references, functions lists...  
+* Release notes  
+* Developer guide
+
+### Elasticsearch
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **explore-analyze**: query languages, scripting, machine learning (NLP)
+* **deploy-manage**: deploying elastic, production guidance, security, and Management  
+* **manage-data**: data store, migration, data lifecycle  
+* **solutions**: core search, playground, hybrid search  
+* **troubleshooting**: troubleshooting pages
+
+#### What is staying in the Elasticsearch repo?
+
+* Reference content (anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Contribution guides and extending elasticsearch (clients, etc)  
+* Release notes
+
+### Cloud
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **deploy-manage**: deploying cloud
+* **cloud-account**: managing your cloud organization  
+* **troubleshooting**: troubleshooting pages
+
+#### **What is staying in the Cloud repo?**
+
+* Reference content (= anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Release notes
+
+### Machine learning
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **deploy-manage**: deploying cloud, managing your cloud organization  
+* **troubleshooting**: troubleshooting pages
+
+#### What is staying in the Elasticsearch repo?
+
+* Reference content, anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Release notes
+
+## How docs sections are organized
+
+The following section shows a mapping of the different areas in the docs IA to the product areas and topics.
+
+| **Area** | **Description** | **Sources (Area / Topic)** |
```

Comment:
is "sources" intended to map old content to new content?

Reply:
consider seeing what you can pull out of [here](https://docs.google.com/document/d/1Omk1x4H62ygJVblhVM-GMybLgdFz-YKGiMQrXE8JqFg/edit?tab=t.0#heading=h.5gzgse23um0u)